get-started/index.md

```diff
@@ -62,7 +65,291 @@ the specifics of how to deploy your Cryostat instance. Continue to [Setup](#setu
 Note: Alternative methods for installing the operator are described in [Alternate Installation Options](/alternate-installation-options) (not recommended).
 ## [Setup](#setup)
 
-### Deploying Cryostat
+### [Configuring Applications](#configuring-applications)
+The following sections will briefly describe how to configure your Java applications so that Cryostat is able to discover and monitor them.
+These examples will assume the application is built with Maven, packaged into an image with a `Dockerfile`, and running in OpenShift,
+but the instructions will be similar for other toolchains and platforms as well.
+
+#### [Using the Cryostat Agent](#using-the-cryostat-agent)
+
+[The Cryostat Agent](/guides/#using-the-cryostat-agent)
+is compatible with Cryostat versions 2.3.0 and newer, and application JDKs 11 and newer. If you are using an older version of Cryostat, please upgrade.
+If your application uses a recent version of JDK8 with JFR support, please either upgrade to JDK11+ or continue to the next section to learn how to configure your
+application without the Cryostat Agent.
+
+The Cryostat Agent JAR must be available to your application JVM. The JAR asset can be downloaded [directly from upstream](https://github.com/cryostatio/cryostat-agent/releases),
+or you may use the following snippet in your `pom.xml` to streamline this.
+
+```xml
+<project>
+  ...
+  <repositories>
+    <repository>
+      <id>github</id>
+      <url>https://maven.pkg.github.com/cryostatio/cryostat-agent</url>
```

Comment:
Is this a public repo that users can freely pull this artifacts from? I rmb there was an issue with authentication.

Reply:
Yea, it's annoying that GitHub still doesn't do unauthenticated pulls from their Maven package registry. I'll add a note about it.